rhino/src/main/java/org/mozilla/javascript/Context.java

```diff
@@ -2327,11 +2327,8 @@ public boolean hasFeature(int featureIndex) {
      */
     @Deprecated
     public XMLLib.Factory getE4xImplementationFactory() {
-        XMLLoader loader = ScriptRuntime.loadOneServiceImplementation(XMLLoader.class);
-        if (loader != null) {
-            return loader.getFactory();
-        }
-        return null;
+        throw new UnsupportedOperationException(
+                "getE4xImplementationFactory is no longer supported");
```

Comment:
What is the intention of this method? It can only provide the implementation class name.
It would help, if I can remove this completely (as it can only provide the implementation class name)
According to javadoc, it is experimental